src/app/modules/admin-shared/components/input/input.component.scss

```diff
@@ -1,4 +1,12 @@
 .custom-form {
     height: 50px;
     border: 1px solid;
+}
+
+.full {
```

Comment:
Please remove this class if not required

Reply:
removed